app/src/main/java/ch/blinkenlights/android/vanilla/PlaylistObserver.java

```diff
@@ -605,6 +625,12 @@ private FileObserver getFileObserver(File target) {
 		return new FileObserver(target.getAbsolutePath(), mask) {
 			@Override
 			public void onEvent(int event, String dirent) {
+				// Events such as IN_IGNORED pass dirent = null.
+				if (dirent == null) {
+					XT("FileObserver::onEvent dirent=null with event " + event);
+					return;
```

Comment:
oh, does that ever happen? Never saw this so far.

But i think we can remove the log-call

Reply:
I saw this throwing NPEs within logcat whenever IN_IGNORED events came in (likely from the watch being unregistered).
It looks like a couple of the Android framework classes rely on IN_IGNORED being received, but it isn't documented from what I understand.
The debug logger has been removed.